mismi-s3/src/Mismi/S3/Commands.hs

```diff
@@ -384,7 +384,7 @@ getObjectsRecursively (Address (Bucket b) (Key ky)) =
         else
           pure $ resp ^. A.lorsContents
 
--- Pair of list of prefixes and list of keys
+-- | Return a tuple of the directories and files at the provided S3 Address.
```

Comment:
Can we leave the wording here as `prefixes` and `keys`, a lot easier to understand in terms of s3